_data/store/helsinkisystems-budgetwatcher.yaml

```diff
@@ -3,7 +3,7 @@ icon: fas fa-keyboard
 demo: false
 gumroad: BudgetWatcher
 subscription: 99 €
-lemonsqueezy: "https://helsinkisystems.lemonsqueezy.com/checkout/buy/a1e2fab6-8563-44c9-ad62-903bf7179de4"
+lemonsqueezy: "https://helsinkisystems.lemonsqueezy.com/checkout?cart=bcc374ec-3bff-4019-ba54-3cdc41587375"
```

Comment:
Sure this is the correct way for sharing the URL?
That is a cart ID and not a product ID

Reply:
Ah yeah, thank you! That not the correct link